### ISSUE TITLE
pytest.mark.no_cover not working as expected

### ISSUE THREAD

**Issue by nlgranger** [pytest-dev/pytest-cov#219]
Hello,

I don't know if this is an issue with my understanding of the doc or with the functionality, but I'm having troubles using `pytest.mark.no_cover`.
I have a test which involves timing and that must run as fast as possible. I'd like to disable coverage during the execution of that test, so I decorated the test with `pytest.mark.no_cover` (btw it is mark and not marker as written in the doc).
Then I ran my test with:
```sh
pytest --cov="seqtools" --cov-config="setup.cfg" tests/test_evaluation.py::test_prefetch_timing
```
The test fails to terminate in time and the coverage report is not 0% as I expected.

Could you please enlighten me on this issue?

**Comment by nlgranger** [comment]
My bad, it looks like the online documentation corresponds to the development branch and not the release, which is confusing because it's written 2.5.1 on the upper left corner...

**Comment by ionelmc** [comment]
Was released in 2.6.0
